src/particle.c

```diff
@@ -140,6 +142,42 @@ void particles_remove_all(void){
 	Nmax 		= 0;
 	N_active 	= -1;
 	N_megno 	= 0;
+	lastID 		= 0;
 	free(particles);
 	particles 	= NULL;
 }
+
+int particles_remove(int index, int keepSorted){
+	if (N==1){
+		printf("Last particle removed.  Exiting.\n");
+		exit(0);
```

Comment:
This has been in there for a while. It might be problematic if this happens within librebound. I don't think anything bad would happen if we do not exit here...